cmd/vcluster/cmd/start.go

```diff
@@ -104,6 +106,7 @@ func NewStartCommand() *cobra.Command {
 
 	cmd.Flags().StringSliceVar(&options.TranslateImages, "translate-image", []string{}, "Translates image names from the virtual pod to the physical pod (e.g. coredns/coredns=mirror.io/coredns/coredns)")
 	cmd.Flags().BoolVar(&options.EnforceNodeSelector, "enforce-node-selector", true, "If enabled and --node-selector is set then the virtual cluster will ensure that no pods are scheduled outside of the node selector")
+	cmd.Flags().StringSliceVar(&options.Tolerations, "toleration", []string{}, "If set will apply the provided tolerations to all pods in the vcluster")
```

Comment:
Maybe we should call this `enforce-toleration` instead, as this will make sure the toleration is set.

Reply:
Makes sense, and it's consistent with node selector 👍